### PR TITLE
Implement noverify opt out for TLS syslog drains

### PR DESCRIPTION
**Pull request by srid**
Configuration is done as part of the drain URI, for example:

    syslog+tls://myserver.com:8583/#insecure=true

Values other than "true" to the "insecure" URI fragment are implicitly treated
as a False.

### CODE REVIEW

**Comment by apg** [comment]
You've broken connect_opts/0 here. But since this is new code anyway, :sheep: it.

**Comment by voidlock** [comment]
:shipit:

**Comment by srid** [comment]
🍷

File: src/logplex_tlssyslog_drain.erl [hunk 1]
```diff
@@ -84,20 +87,44 @@
...
+                  authority=#ex_uri_authority{host=Host, port=Port},
+                  fragment=Fragment}) ->
+    Opts =  parse_fragment(Fragment),
+    Insecure = case proplists:lookup("insecure", Opts) of
+                   {"insecure", "true"} -> true;
+                   _ -> false
+               end,
+    {Host, Port, Insecure}.
+
+parse_fragment(undefined) ->
```
Comment (archaelus) [inline src/logplex_tlssyslog_drain.erl]: This seems like a lot of work to parse the fragment - there's no particular standard for the encoding in the fragment, so you could just match `Fragment =:= "#insecure"` or `Fragment =:= "insecure=true"` here right? (If the 'insecure' thing isn't set in stone, you could also use something like 'skip_cert_validation' which would require less devcenter documentation to explain the purpose of this option).

**Pull request merged**